internal/pkg/print/print.go

```diff
@@ -85,3 +91,27 @@ func (p *Printer) Error(msg string) {
 func (p *Printer) OutOrStdout() io.Writer {
 	return p.Cmd.OutOrStdout()
 }
+
+// Prompts the user for confirmation.
+//
+// Returns nil only if the user (explicitly) answers positive.
+// Returns ErrAborted if the user answers negative.
+func (p *Printer) PromptForConfirmation(cmd *cobra.Command, prompt string) error {
+	question := fmt.Sprintf("%s [y/N] ", prompt)
+	reader := bufio.NewReader(cmd.InOrStdin())
```

Comment:
Would it make sense to have a Print.InOrStdin() for this? Don't really see a use case for this now, but could be useful in the future

Reply:
Maybe, or we could also just call `p.Cmd.InOrStdin()` (and do the same for `p.Cmd.InOrStdin()`, removing `(p *Printer) OutOrStdout()`)

Reply:
I think you meant `p.Cmd.InOrStdout` the second time @joaopalet, but I agree, it seems unnecessary to have those methods defined in the `print` pkg

Reply:
Yes, calling `p.Cmd.InOrStdin()` makes sense. 
However, I don't understand the removing the `(p *Printer) OutOrStdout()`.

Reply:
But removing `p.OutOrStdout()` would mean that the `pager.Display` method, from the pager package, would have to receive a cmd in arguments. 
I think it makes more sense to keep this method than change the Display signature

Reply:
But in `pager.Display` we could also do `p.Cmd.OutOrStdout`, right?